var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -90,9 +90,13 @@ def configure_args(self):
             '-des',
             '-Dprefix={0}'.format(prefix),
             '-Dlocincpth=' + self.spec['gdbm'].prefix.include,
-            '-Dloclibpth=' + self.spec['gdbm'].prefix.lib,
+            '-Dloclibpth=' + self.spec['gdbm'].prefix.lib
         ]
 
+        # Prepend default perl @INC path to allow package activation:
+        config_args.append('-Accflags=-DAPPLLIB_EXP=\\"' + join_path(
+                           self.spec.prefix, 'lib', 'perl5') + '\\"')
```

Comment:
I have no objection about making things cleaner, but the search path must literally be `.../lib/perl5`. Are you sure that `self.prefix.lib` is always `.../lib` and never something like `.../lib64`?